Description: let it probe for other filesystems
 Not having a partition does not constitute a bad device.
Author: Milan Kupcevic <milan@debian.org>
Bug-Debian: http://bugs.debian.org/819549
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/second/fs_swap.c
+++ b/second/fs_swap.c
@@ -60,8 +60,8 @@
      DEBUG_OPEN;
 
      if (file->device_kind != FILE_DEVICE_BLOCK || part == NULL) {
-	  DEBUG_LEAVE(FILE_ERR_BADDEV);
-	  return FILE_ERR_BADDEV;
+          DEBUG_LEAVE(FILE_ERR_BAD_FSYS);
+          return FILE_ERR_BAD_FSYS;
      }
 
      /* We assume that device is "short" and is correctly NULL terminsated */
